README.md

```diff
@@ -1,2 +1,14 @@
 # ros_install
 Repo that contains all of the rosinstall file
+
+
+## Add packages
+
+To add a packages to the rosinstall just add your packages to the following files: desktop.rosinstall and sara.rosinstall
```

Comment:
change for [...] files: [desktop.rosinstall](desktop.rosinstall) and [sara.rosinstall](sara.rosinstall)